src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -394,6 +397,10 @@ public function preprocessUitpasPromotions(\CultureFeed_ResultSet $resultSet)
         return $promotions;
     }
 
+    public function formatOrganizerUrlName($organizerName)
```

Comment:
Also, and I just noticed this after it was merged, but adding type-hints to methods would be a quick win to gradually introduce them to this codebase now that we're on PHP 7+
Fine for now though.